themes/root/templates/HelpTranslations/it/tag.phtml

```diff
@@ -11,4 +11,4 @@ I tag sono generalmente scelti in modo informale e personalmente dall'autore/cre
 <p>
 In genere, un elemento avrà una o più & quot; tags & quot; ad esso associati, come parte di alcuni software o del sistema di classificazione. Il software fornisce collegamenti ad altri elementi che condividono quel tag parola chiave, o anche a collezioni di tag specifici. Ciò permette percorsi &quot; multi-navigabili&quot; attraverso gli elementi che possono rapidamente e facilmente essere modificati dall'amministratore della collezione, con il minimo sforzo.</p>
```

Comment:
Looks like there are malformed entities in this one, though that's apparently a pre-existing condition.